### ISSUE TITLE
[BUG] UI disappears after sleeping 

### ISSUE THREAD

**Issue by catweaselirl** [r4isen1920/AdvancementPack#2]
Title. 

Enable pack -> sleep -> UI will be gone completely until game restart

**Comment by JangoBritt** [comment]
Unfortunately this issue still exists, making the mod unplayable until it's resolved.

**Comment by CrispPlate** [comment]
was is solved already?

**Comment by catweaselirl** [comment]
> was is solved already?

No. If you're technically inclined you can view the source code and attempt to fix it yourself
